helpdesk/views/staff.py

```diff
@@ -422,8 +423,16 @@ def view_ticket(request, ticket_id):
 
         return redirect('helpdesk:edit_ticket_checklist', ticket.id, checklist.id)
 
+    # List open tickets on top
+    dependencies = ticket.ticketdependency.annotate(
+        rank=Case(
+                When(depends_on__status__in=Ticket.OPEN_STATUSES, then=Value('1')),
+                default=Value('2')
```

Comment:
I believe you can use integer for the rank values

```suggestion
                When(depends_on__status__in=Ticket.OPEN_STATUSES, then=Value(1)),
                default=Value(2)
```